client/src/components/pages/SignupPage.jsx/SignupPage.jsx

```diff
@@ -1,21 +1,210 @@
+                  <label>Username</label>
+                  </div>
+                  <div className="form__input">
+                  <input
+                    
+                    name="username"
+                    
+                    value={values.username}
+                    onChange={handleChange}
+                    onBlur={handleBlur}
+                    className={
+                      errors.username && touched.username ? "input-error" : null
+                    }
+                  />
+                  </div>
+                  {errors.username && touched.username && (
+                    <div className="error">{errors.username}</div>
+                  )}
+                </div>
+  
+                <div className="form-row">
+                  <div className="form__label">
+                  <label htmlFor="password">Password</label>
+                  </div>
+                  <div className="form__input">
+                  <input
+                    type="password"
+                    name="password"
+                    id="password"
+                    value={values.password}
+                    onChange={handleChange}
+                    onBlur={handleBlur}
+                    className={
+                      errors.password && touched.password ? "input-error" : null
+                    }
+                  />
+                  </div>
+                
+                  {errors.password && touched.password && (
+                    <div className="error">{errors.password}</div>
+                  )}
+                </div>
+                <div className="form-row">
+                  <div className="form__label">
+                  <label htmlFor="confirm-password">Confirm Password</label>
+                  </div>
+                  <div className="form__input">
+                  <input
+                    type="password"
+                    name="confirmPassword"
+                    id="confirmPassword"
+                    value={values.confirmPassword}
+                    onChange={handleChange}
+                    onBlur={handleBlur}
+                    className={
+                      errors.confirmPassword && touched.confirmPassword ? "input-error" : null
+                    }
+                  />
+                  </div>
+                
+                  {errors.confirmPassword && touched.confirmPassword && (
+                    <div className="error">{errors.confirmPassword}</div>
+                  )}
+                </div>
+                <div className="form__submit">
+                <Button
+                  type="submit"
+                  className={!(dirty && isValid) ? "disabled-btn" : ""}
+                  disabled={!(dirty && isValid)}
+                >
+                  Sign up
+                </Button>
+                <div className="bottom-text">
+                    Already have an account? <Link to="/auth/signup">Login </Link> here
```

Comment:
Should go to the login page. Incorrect Route

`<Link to="/auth/login">Login</Link>`